core-web/apps/dotcms-ui/src/app/view/components/dot-listing-data-table/dot-listing-data-table.component.html

```diff
@@ -123,6 +123,7 @@
     </td>
     <td class="listing-datatable__action-column" style="width: 5%">
         <dot-action-menu-button
+            *ngIf="actions"
```

Comment:
why ?

Reply:
Angular gets stuck in an error loop, if no actions provided to the table